app/assets/javascripts/jsxc.js

```diff
@@ -42,6 +42,12 @@ $(document).ready(function() {
             onlogin: true
           }
         });
+
+        // auto login if the user hasn't disabled it
+        var onLogin = !(app.currentUser.get("disable_chat_login"));
```

Comment:
How about using `chat_auto_login` or something similar everywhere? (jsxc.js, presenter, controller, database... default: `true`) I think it is easier to understand if you describe the “positive” event.